packages/uglify/README.md

```diff
@@ -8,16 +8,15 @@ This block provides UglifyJS webpack plugin configuration for [webpack-blocks](h
 
 Based on [uglifyjs-webpack-plugin](https://github.com/webpack-contrib/uglifyjs-webpack-plugin) (not `webpack.optimize.UglifyJsPlugin`) which uses UglifyJS v3 (uglify-es) that supports ECMAScript 2015.
 
+**This block will only work in the `production` mode.**
```

Comment:
Idea: Let’s turn `mode` into a link to the `setMode` docs.

Reply:
In the other PR.